production/helm/loki/templates/query-scheduler/service-query-scheduler.yaml

```diff
@@ -21,7 +21,7 @@ spec:
   ports:
     - name: http-metrics
       port: 3100
-      targetPort: http
+      targetPort: http-metrics
```

Comment:
Do we have the same issue for service-query-frontend-headless.yaml?

Reply:
https://github.com/grafana/loki/commit/1ab9d271c354caf0ba589691e6477fb9a19039f0 looks like it was fixed just recently